tribuo-classification/gradle.properties

```diff
@@ -1,5 +1,2 @@
-#
-#Tue Feb 09 22:52:29 GMT 2021
-conclaveVersion=1.1
+conclaveVersion=1.2-RC1
```

Comment:
We've just released RC2

Reply:
Before merging to master, this will be 1.2. 
Anyways I will test this again once we have a final release candidate.